python-avd/pyavd/_eos_designs/schema/schema_fragments/defs_node_type.schema.yml

```
@@ -212,7 +212,7 @@ $defs:
             type: list
             items:
               type: str
-              pattern: "Ethernet[\\d/]+"
+              pattern: "Ethernet\\d+(-\\d+)?(,\\d+(-\\d+)?)*(/\\d+(-\\d+)?(,\\d+(-\\d+)?)*)*"
```

NOTE: This pattern must be changed. Key description claims support for the interface ranges. Current pattern ` "Ethernet[\\d/]+"` will not match any range

---

This will not match `"Ethernet1, Ethernet3"` which is also allowed. Let's simplify the pattern for now and then revisit later.

Not sure if this will work:
```suggestion
              pattern: "Ethernet[\\d/,(Ethernet) -]+"
```

---

we do support the ranges here. We do in many places. We validate the ranges intensively in the expansion logic, so maybe we could just do `Ethernet.*` for now.

---

`range_expand` seems to not enforce any strict syntax or patterns

```
>>> from pyavd.j2filters import range_expand
>>> range_expand("Ethernet1-")
['Ethernet1']
>>> range_expand("Ethernet1-2/")
['Ethernet1', 'Ethernet2']
>>> range_expand("Abc1-2/")
['Abc1', 'Abc2']
>>>
```

---

Relaxed this patterns to `Ethernet.*`